### ISSUE TITLE
The object destructuring here is not supported in IE 11 and is breaking. How is this package being compiled?

### ISSUE THREAD

**Issue by musicpulpite** [missive/emoji-mart#339]
https://github.com/missive/emoji-mart/blob/f654ef4050dd8fe4e06af96818bbddebcf8c4650/src/utils/emoji-index/emoji-index.js#L5

**Comment by nolanlawson** [comment]
Yeah, it looks like our use of babel is not transpiling everything to pure ES5: https://unpkg.com/emoji-mart@2.11.1/dist/utils/emoji-index/emoji-index.js

In the meantime, you can configure your Webpack to run Babel on emoji-mart if you need IE11 support.

**Comment by musicpulpite** [comment]
Yeah. Unfortunately my company is still using Meteor so it took a little more hacking in order to expose it to the compiler/bundler. Haha. Just so long as you are aware of the issue.

**Comment by nolanlawson** [comment]
emoji-mart uses a lot of conventions that don't work in IE (e.g. `Object.values()`). I'm not sure if we ever explicitly targeted IE support – it could just be that emoji-mart does not support IE at all.

According to Wikipedia, [IE market share is at 2-3% and dropping](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers#Summary_tables). I'm not sure if anyone will get around to IE support before IE becomes obsolete. :thinking:

**Comment by shabanzo** [comment]
Btw, I already use Babel on emoji-mart but get this error:
![IE11 Issue](https://user-images.githubusercontent.com/8907660/74653968-defbce80-51bb-11ea-80fb-189419e1374e.png)
![IE11 Issue 2](https://user-images.githubusercontent.com/8907660/74653976-e15e2880-51bb-11ea-8051-ca7803568d99.png)

**Comment by EtienneLem** [comment]
Given IE11 has been retired by Microsoft, we do no plan on supporting it.
